engines/dragons/background.cpp

```diff
@@ -395,7 +395,7 @@ uint16 ScaleLayer::getScale(uint16 y) {
 			if (local_v0_368 != 0) {
 				iVar3 = ((uVar5 & 0xffffu) - (uVar7 & 0xffffu)) * (uint)(uint16)(y - pSVar6->_y);
 
-				assert(((uint16)local_v0_368 != 0xffffu) || (iVar3 != (int)-0x80000000));
+				assert(((uint16)local_v0_368 != 0xffffu) || (iVar3 != (-1 - 0x7fffffff)));
```

Comment:
Maybe it will be more readable to cast iVar3 to unit and make that comparison?

Reply:
Changed iVar3 to uVar3 and retyped as uint instead